price-feeder.example.toml

```diff
@@ -1,15 +1,19 @@
-gas_adjustment = 1
+gas_adjustment = 1.5
 provider_timeout = "1000000s"
 [server]
 listen_addr = "0.0.0.0:7171"
 read_timeout = "20s"
-verbose_cors = true
+verbose_cors = false
```

Comment:
I see these on facu's PR for umee (#208) - can you comment on why we need to change these? Also, why is USDC/USD from kraken added here?

This will definitely conflict with #195

Reply:
Changed this back. I had tried it as a solution when the e2e wasn't working but that didn't end up being the solution (forgot to revert it back). I know on umee the gas adjustment needing to be higher was something we found a couple months ago